homeassistant/components/snapcast/media_player.py

```diff
@@ -114,166 +119,233 @@ class SnapcastGroupDevice(MediaPlayerEntity):
         | MediaPlayerEntityFeature.SELECT_SOURCE
     )
 
-    def __init__(self, group, uid_part, entry_id):
-        """Initialize the Snapcast group device."""
+    def __init__(self, device, server, entry_id) -> None:
+        """Initialize the base device."""
         self._attr_available = True
-        self._group = group
+        self._device = device
+        self._server = server
         self._entry_id = entry_id
-        self._attr_unique_id = f"{GROUP_PREFIX}{uid_part}_{self._group.identifier}"
+        self._attr_media_content_type = MediaType.MUSIC
```

Comment:
I understand where you're coming from as the stream could be a broadcast, notification, etc,

But
A. `MediaType.MUSIC` is required for `artist` and `album` metadata to be parsed.
B. Snapcast doesn't currently provide a means to specify this.

Of the available MediaType, `MUSIC` and `PODCAST` might be the only relevant types.

```python
class MediaType(StrEnum):
    """Media type for media player entities."""

    ALBUM = "album"
    APP = "app"
    APPS = "apps"
    ARTIST = "artist"
    CHANNEL = "channel"
    CHANNELS = "channels"
    COMPOSER = "composer"
    CONTRIBUTING_ARTIST = "contributing_artist"
    EPISODE = "episode"
    GAME = "game"
    GENRE = "genre"
    IMAGE = "image"
    MOVIE = "movie"
    MUSIC = "music"
    PLAYLIST = "playlist"
    PODCAST = "podcast"
    SEASON = "season"
    TRACK = "track"
    TVSHOW = "tvshow"
    URL = "url"
    VIDEO = "video"
```

We could attempt to deduce the type by checking the metadata. If an `album` or `artist` property is present set the type to `MUSIC` otherwise leave it empty.

Reply:
Good point, didn't know that. For me both solutions are fine.
Maybe it is easier then to set it to Music as you suggest.

Reply:
Unless anyone else feels strongly I would prefer to leave this set to Music.